# Урок 4
## Задания:

:x: 1. Придумать класс, методы которого могут завершаться неудачей и возвращать либо значение, либо ошибку Error?. Реализовать их вызов и обработать результат метода при помощи конструкции if let, или guard let. <br/>

:x: 2. Придумать класс, методы которого могут выбрасывать ошибки. Реализуйте несколько throws-функций. Вызовите их и обработайте результат вызова при помощи конструкции try/catch. <br/>


* :x: - не начато
* :white_check_mark: - готово
* :memo: - в работе
* :warning: - требуются исправления
